### ISSUE TITLE
Is there any option to drop cdc-field on the schema of sink target table ?

### ISSUE THREAD

**Issue by okayhooni** [tabular-io/iceberg-kafka-connect#229]
I think it looks more pretty cdc-field(with value of `I` & `U`) is dropped on the target table schema.

But, as you know, it must not drop on SMT chain for connector to processing CDC topic message as expected.

Is there any option to drop cdc-field on the schema of sink target table ?

**Comment by fqtab** [comment]
This is not possible today. 
We might consider adding such a feature in the future but this is relatively low priority. 

I'm not 100% sure if this will work but you can try it and report back if it does/doesn't work: 
- Either precreate the table without the cdc-field before starting the connector
- Or delete the cdc-field from the table after starting the connector

**Comment by okayhooni** [comment]
Thank you for kind response!

I will close this issue, event though I couldn't apply your suggestion, because I want to sink with `iceberg.tables.evolve-schema-enabled` option..!
